docs/deployment.md

```diff
@@ -1,66 +1,182 @@
 # Deployment
 
-One of the key features of BigFlow CLI is the full automation of the build and deployment process.
-BigFlow can [build](project_setup_and_build.md) your workflows
-to dockerized Airflow DAGs and deploy them to Google Cloud Composer.
+One of the key features of BigFlow is the full automation of the build and deployment process.
+BigFlow can dockerize your workflows and deploy them to Google Cloud Composer.
 
-## Deployment artifacts
-There are two deployment artifacts:
+## GCP runtime environment
+
+BigFlow GCP runtime environment consists of two services:
+
+1. Google [Cloud Composer](#cloud-composer),
+2. [Docker Registry](#docker-registry).
+
+Typically, for one software project, teams use one or more
```

Comment:
To że to długo trwa to jedno, a drugie to że jest jeden artefakt, niezależny od środowiska.